package.json

```diff
@@ -3,13 +3,21 @@
   "version": "0.1.0",
   "devDependencies": {
     "grunt": "~0.4.1",
+    "grunt-closure-linter": "~0.1.0",
+    "grunt-closure-tools": "~0.8.3",
+    "grunt-contrib-concat": "~0.3.0",
     "grunt-contrib-jshint": "~0.6.0",
     "grunt-contrib-nodeunit": "~0.2.0",
-    "grunt-contrib-uglify": "~0.2.2",
     "grunt-contrib-qunit": "~0.2.2",
+    "grunt-contrib-uglify": "~0.2.2",
     "grunt-contrib-watch": "~0.4.4",
-    "grunt-contrib-concat": "~0.3.0",
-    "grunt-closure-tools": "~0.8.3",
-    "grunt-closure-linter": "~0.1.0"
+    "jasmine": "^3.5.0",
```

Comment:
We should probably only stick to patch releases. Even minor release might create a breaking change and someone will have to go and urgently fix it. 
(here and below)

Reply:
Replaced with ~